2019-11-06 17:16:37,867 - ERROR -Leo : [Errno 2] No such file or directory: '.\\model/2019110514OneClassSVM.pickle'
2019-11-06 17:16:37,867 - ERROR -Leo : [Errno 2] No such file or directory: '.\\model/2019110514PCA.txt'
2019-11-12 14:30:51,544 - WARNING -Leo : scaleModel:找不到對應檔案
2019-11-12 14:30:51,546 - ERROR -Leo : File "c:/acs/ai7000/project/incineratormotor/code/spydervibrated/vibratedvoice_ensemble.py", line 88, in <module>: [AttributeError] 'NoneType' object has no attribute 'transform'
2019-11-12 14:36:04,408 - WARNING -Leo : scaleModel:找不到對應檔案
2019-11-12 14:38:16,485 - ERROR -Leo : File "c:/acs/ai7000/project/incineratormotor/code/spydervibrated/vibratedvoice_ensemble.py", line 88, in <module>: [AttributeError] 'NoneType' object has no attribute 'transform'
2019-11-12 14:43:07,866 - WARNING -Leo : scaleModel:could not find correspond file
2019-11-12 14:43:24,227 - ERROR -Leo : File "c:/acs/ai7000/project/incineratormotor/code/spydervibrated/vibratedvoice_ensemble.py", line 88, in <module>: [AttributeError] 'NoneType' object has no attribute 'transform'
2019-11-12 14:56:04,943 - WARNING -Leo : .\model: could not find correspond file
2019-11-12 14:56:04,944 - ERROR -Leo : File "c:/acs/ai7000/project/incineratormotor/code/spydervibrated/vibratedvoice_ensemble.py", line 88, in <module>: [AttributeError] 'NoneType' object has no attribute 'transform'
2019-11-12 15:17:21,982 - WARNING -Leo : .\model/scaleModel.pickle: could not find correspond file
2019-11-12 15:17:21,983 - ERROR -Leo : File "c:/acs/ai7000/project/incineratormotor/code/spydervibrated/vibratedvoice_ensemble.py", line 88, in <module>: [AttributeError] 'NoneType' object has no attribute 'transform'
2019-11-12 15:26:21,196 - WARNING -Leo : .\model\scaleModel.pickle: could not find correspond file
2019-11-12 15:26:21,196 - ERROR -Leo : File "C:/ACS/AI7000/Project/IncineratorMotor/code/spyderVibrated/vibratedVoice_ensemble.py", line 88, in <module>: [AttributeError] 'NoneType' object has no attribute 'transform'
2019-11-12 15:33:13,633 - ERROR -Leo : File "C:\ACS\AI7000\Project\IncineratorMotor\code\spyderVibrated\inference\inferencePCA.py", line 22, in inferencePCA: [FileNotFoundError] 2
2019-11-12 15:39:01,274 - ERROR -Leo : File "C:\ACS\AI7000\Project\IncineratorMotor\code\spyderVibrated\inference\inferencePCA.py", line 22, in inferencePCA: [FileNotFoundError] 2
2019-11-12 15:42:53,471 - WARNING -Leo : ./model/2019110811OneClassSVM:file not exists
2019-11-12 15:43:27,100 - WARNING -Leo : ./model/2019111215OneClassSVM:file not exists
2019-11-12 16:06:21,074 - WARNING -Leo : ./model\2019110811OneClassSVM.pickle: could not find correspond file
2019-11-12 16:06:21,075 - ERROR -Leo : File "C:\ACS\AI7000\Project\IncineratorMotor\code\spyderVibrated\inference\inferenceEnsemble.py", line 26, in inferenceEnsemble: [FileNotFoundError] 2
